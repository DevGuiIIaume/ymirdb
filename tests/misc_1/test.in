set $#%*#$T#$ 1 2 3
set MoreThanMaxKeySizeeeeeeee 1 2 3
set 7ismyfavouritenumber -1 0 1
set is_this_key_ok 1 2 3
bye
