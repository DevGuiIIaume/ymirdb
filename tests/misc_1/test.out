> not permitted

> not permitted

> not permitted

> not permitted

> bye
